docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
 
-    neptune_logger = NeptuneLogger(
-        api_key="ANONYMOUS",
-        project_name="shared/pytorch-lightning-integration",
-        close_after_fit=False,
-        ...
-    )
+    pip install pytorch-lightning torchvision
 
-**Log test metrics**
+From Conda:
 
-.. code-block:: python3
+.. code-block:: bash
+
+    conda install pytorch-lightning -c conda-forge
+    conda install pytorch torchvision -c pytorch
 
-    trainer.test(model)
+Now, you have all dependencies installed. Let's move to the actual integration.
 
-**Log additional metrics**
+Step 1 - Necessary Imports
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+No a big deal here, just imports. Note ``pytorch_lightning`` at the bottom.
 
 .. code-block:: python3
 
-    from sklearn.metrics import accuracy_score
-    ...
-    accuracy = accuracy_score(y_true, y_pred)
+    import os
+
+    import torch
+    from torch.nn import functional as F
+    from torch.utils.data import DataLoader
+    from torchvision.datasets import MNIST
+    from torchvision import transforms
 
-    neptune_logger.experiment.log_metric('test_accuracy', accuracy)
+    import pytorch_lightning as pl
 
-**Log performance charts**
+Step 2 - Model Hyper-Parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+You will see them in Neptune parameters tab.
 
 .. code-block:: python3
 
-    from scikitplot.metrics import plot_confusion_matrix
-    import matplotlib.pyplot as plt
-    ...
-    fig, ax = plt.subplots(figsize=(16, 12))
-    plot_confusion_matrix(y_true, y_pred, ax=ax)
+    PARAMS = {'max_epochs': 3,
+              'LR': 0.02,
+              'batch_size': 32}
 
-    neptune_logger.experiment.log_image('confusion_matrix', fig)
+Step 3 - Define Lightning Module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This is minimal example of the ``pl.LightningModule``. Notice the Cross Entropy loss in the ``training_step`` method.
 
-**Save checkpoints folder after training**
+Also, note that you pass learning rate from the ``PARAMS`` dictionary.
 
 .. code-block:: python3
 
-    model_checkpoint = pl.callbacks.ModelCheckpoint(filepath='my/checkpoints')
+    class LitModel(pl.LightningModule):
+        def __init__(self):
+            super().__init__()
+            self.l1 = torch.nn.Linear(28 * 28, 10)
 
-    trainer = Trainer(logger=neptune_logger,
-                      checkpoint_callback=model_checkpoint)
-    trainer.fit(model)
+        def forward(self, x):
+            return torch.relu(self.l1(x.view(x.size(0), -1)))
 
-    neptune_logger.experiment.log_artifact('my/checkpoints')
+        def training_step(self, batch, batch_idx):
+            x, y = batch
+            y_hat = self(x)
+            loss = F.cross_entropy(y_hat, y)
+            return pl.TrainResult(loss)
 
-**Explicitly close the logger**
+        def configure_optimizers(self):
+            return torch.optim.Adam(self.parameters(), lr=PARAMS['LR'])
 
-Closing is optional but you may want to close it and then do something afterwards.
+Step 4 - Prepare Data Loader
```

Comment:
Those two steps feel like one step to me in this context. 

We are teaching people how to do Neputne not lightning so I'd put Loader and System in one Step for sure.

Add two substeps of, course. 
1. 
2.

Reply:
agreed.